pkg/config/config.go

```diff
@@ -47,6 +47,7 @@ const (
 	MasterContainerName = "kube-sonobuoy"
 	// MasterResultsPath is the location in the main container of the master pod where results will be archived.
 	MasterResultsPath = "/tmp/sonobuoy"
+	DefaultWaitOutput = "silent"
```

Comment:
Once a type is in place so we can properly validate this flag whenever set, this would be changed to referring to a specific, typed value not the string.